drivers/nutdrv_qx_voltronic-qs-hex.c

```diff
@@ -75,6 +75,7 @@ static item_t	voltronic_qs_hex_qx2nut[] = {
 	/* Query UPS for status
 	 * > [QS\r]
 	 * < [#6C01 35 6C01 35 03 519A 1312D0 E6 1E 00001001\r]	(after being preprocessed)
+	 *   [#6901 6c 6802 6c 00 5fd7 12c000 e4 1e 00001001 02] (this is the answer of the LYONN CTB-800v)
```

Comment:
The answer should be CR-terminated (and I'd prefer the following bracket-enclosed comment to be separated by a tab and the comment on the previous line to be aligned with it):

```
	 * < [#6C01 35 6C01 35 03 519A 1312D0 E6 1E 00001001\r]		(after being preprocessed)
	 *   [#6901 6c 6802 6c 00 5fd7 12c000 e4 1e 00001001 02\r]	(this is the answer of the LYONN CTB-800v)
```